<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main widget background */
        QWidget#MainWidget {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #2c3e50, stop:1 #34495e);
        }

        QLabel#Title {
            color: #ecf0f1;
            background: transparent;
            padding: 10px;
            border-radius: 8px;
        }

        QComboBox {
            background-color: #ffffff;
            border: 2px solid #bdc3c7;
            border-radius: 6px;
            padding: 8px;
            font-size: 14px;
            color: #2c3e50;
            min-height: 20px;
        }

        QComboBox:focus {
            border: 4px solid #00e5ff;
            color: #00e5ff;
            font-size: 14px;
            border-radius: 8px;
        }

        QPushButton#PlayButton {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #2ecc71, stop:1 #27ae60);
            color: white;
            border: none;
            border-radius: 8px;
            padding: 12px;
            font-size: 14px;
            font-weight: bold;
        }

        QPushButton#PlayButton:hover {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #27ae60, stop:1 #229954);
        }

        QPushButton#PlayButton:pressed {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #229954, stop:1 #1e8449);
        }

        QPushButton#PlayButton:focus {
            border: 4px solid #00e5ff;
            color: #00e5ff;
            font-size: 14px;
            border-radius: 8px;
        }

        QPushButton#ExitButton {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #e74c3c, stop:1 #c0392b);
            color: white;
            border: none;
            border-radius: 8px;
            padding: 12px;
            font-size: 14px;
            font-weight: bold;
        }

        QPushButton#ExitButton:focus {
            border: 4px solid #00e5ff;
            color: #00e5ff;
            font-size: 14px;
            border-radius: 8px;
        }

        QPushButton#ExitButton:hover {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #c0392b, stop:1 #a93226);
        }

        QPushButton#ExitButton:pressed {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
            stop:0 #a93226, stop:1 #922b21);
    }</string>
  </property>
  <widget class="QFrame" name="Menu">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Title">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>BrickGame</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="SelectGameBox">
      <property name="placeholderText">
       <string>select a game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PlayButton">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>SelectGameBox</tabstop>
  <tabstop>PlayButton</tabstop>
  <tabstop>ExitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
